kitchenci/.kitchen.yml

```diff
@@ -0,0 +1,16 @@
+---
+driver:
+  name: docker
+
+provisioner:
+  name: puppet_apply
+  puppet_version: "3.8.1"
+  mainfests_path: manifests
+  modules_path: modules
+
+platforms:
+  {platform-placeholder}
```

Comment:
well the distribution parameter in jenkins job will be a text with a pre-defined syntax e.x.

-name: centos-6.6
-name: ubuntu-14.04
and replacing the {platform-placeholder} with the value of distribution parameter in .kitchen.yml file